### DISCUSSION TITLE
Images on Items

### DISCUSSION THREAD

**Discussion by zipus** [frePPLe/frepple#571] [Ideas]
The product image is a common need for our customers for their ERP, (some business want a customer final image, and some others want a screenshot of a CAD file to be used for the production employees).
As I think that this is not a custom field as its business independent, shouldn't be on frepple?
For me it should display on the item edit section, but also I would find interesting to make a thumbnail for each column that references the item table (see first row, column Item).
![image](https://user-images.githubusercontent.com/1032986/199557128-55a66ad8-574e-474a-9d36-9c1329445f6d.png)

Is the community in need of this feature? Which are your thoughts about it?

**Comment by jdetaeye** [comment]
My 2 cents:
- We have a similar feature with an avatar on the users. You can also upload attachments as comments to any object. In practice, we have seen very few people using either of these functonalities.
- An ERP system has many users from all departments within a company. Frepple as a planning tool only has a handful of users. IMHO, the value of displaying the images is much less in frepple than in an ERP.  And it doesn't justify the effort of uploading all the images.
- A report in frepple can easily display 100 rows with different items. The performance impact of downloading all those thumbnails will be significant.
- In summary, I'm not in favor of doing this as a generic feature.  Best alternative is to support custom attributes of type image. With such an enhancement users can add pictures to any model, and you can choose whether or not display the field in your reports.

**Comment by zipus** [comment]
> Best alternative is to support custom attributes of type image. With such an enhancement users can add pictures to any model, and you can choose whether or not display the field in your reports.

LGTM
Hey there community! Upvote this if you are interested!

**Comment by zipus** [comment]
> Best alternative is to support custom attributes of type image. With such an enhancement users can add pictures to any model, and you can choose whether or not display the field in your reports.

Maybe better or in complement of this the attribute image should allow URL instead of base64, in most cases, ERPs can display the images from some endpoint. That way the load on frepple database would be less, this will also allow lazy-loading.
